app/page-partials/my-dashboard/IntegrationInfoTabs/DigitalCredentialPanel.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import DigitalCredentialStatus from './DigitalCredentialStatus';
+import { Integration } from 'interfaces/Request';
+import { SubTitle, ApprovalContext, ApprovedAndWait, ApprovedAndAvailable } from './shared';
+
+interface Props {
+  integration: Integration;
+  approvalContext: ApprovalContext;
+}
+
+function DigitalCredentialPanel({ integration, approvalContext }: Readonly<Props>) {
+  const {
+    hasProd,
+    hasDigitalCredential,
+    awaitingDigitalCredentialProd,
+    digitalCredentialProdApplying,
+    digitalCredentialApproved,
+  } = approvalContext;
+  if (!hasProd || !hasDigitalCredential) return null;
```

Comment:
so this panel won't display if the dc does not have prod? What if a team is testing with no prod integration?

Reply:
yeah, this additional panel is just showing the status of the production approval steps, it's beside the normal terraform one. So if they don't have prod it can be hidden